### ISSUE TITLE
Quest disappearing

### ISSUE THREAD

**Issue by nelle013** [HabitRPG/habitica#4143]
Twice over the last 24 hours I have invited my party to a quest, some or most of them accept it, then I go to the page to check on who still needs to accept it and it's gone. Its like I aborted it, but I didn't... no warning or anything, the quest invite is gone and the quest is back in my inventory.

**Comment by Alys** [comment]
One of your party members is probably experiencing this intermittent bug: ["The quest owner has left" when they have not - Issue #4074](https://github.com/HabitRPG/habitrpg/issues/4074) and they'll be cancelling the quest. If they reload the page when they see the "quest owner has left" message, the message should disappear and they'll see the participant list as normal. 

You can follow the progress of that bug using the link above. I'll be doing a fix for it in the next day or so.

**Comment by scarlet-lettered** [comment]
My quest has repeatedly disappeared as well.  Members of my party were seeing the "owner has left" message.  But I informed party about the bug, instructed them not to cancel the quest, but the quest disappeared again.  They all claim they did not abort the quest this time.  I've restarted the quest again, and repeated the instructions for avoiding the bug.  But I'm wondering if something more is going on.
